src/aof.c

```diff
@@ -2102,6 +2102,35 @@ int rewriteModuleObject(rio *r, robj *key, robj *o, int dbid) {
     return io.error ? 0 : 1;
 }
 
+static int rewriteFunctions(rio *aof) {
+    dict *functions = functionsLibGet();
+    dictIterator *iter = dictGetIterator(functions);
+    dictEntry *entry = NULL;
+    while ((entry = dictNext(iter))) {
+        functionLibInfo *li = dictGetVal(entry);
+        if (li->desc) {
+            if (rioWrite(aof, "*7\r\n", strlen("*7\r\n")) == 0) goto werr;
+        } else {
+            if (rioWrite(aof, "*5\r\n", strlen("*5\r\n")) == 0) goto werr;
```

Comment:
It seems to be more appropriate to use constant.
The same goes for `strlen("description")` below.
```suggestion
            if (rioWrite(aof, "*7\r\n", 4) == 0) goto werr;
        } else {
            if (rioWrite(aof, "*5\r\n", 4) == 0) goto werr;
```

Reply:
Fixed